reports/README.md

```diff
@@ -0,0 +1,10 @@
+## Authoring krabby reports
```

Comment:
My preference would be to create a `docs/` directory for assorted stuff then move it to the main README or wiki when it's appropriate. It just helps to know that everything is in the same place. That said using `find` is easy and this is just fine too so long as we're aware of it.